products/compute/api.yaml

```diff
@@ -672,6 +672,16 @@ objects:
 
                 For CONNECTION mode, either maxConnections or
                 maxConnectionsPerInstance must be set.
+            - !ruby/object:Api::Type::Integer
+              name: 'maxConnectionsPerEndpoint'
+              description: |
+                The max number of simultaneous connections that a single backend
+                network endpoint can handle. This is used to calculate the
+                capacity of the group. Can be used in either CONNECTION or
+                UTILIZATION balancing modes.
+
+                For CONNECTION mode, either
```

Comment:
The comments for maxConnections and maxConnectionsPerInstance say "For CONNECTION mode, either maxConnections or maxConnectionsPerInstance must be set". I see that the GCP docs say these exact things, but it looks like a contradiction to me- can you check on it?

Reply:
By contradiction, do you mean that if maxConnectionsPerInstance and maxConnectionsPerEndpoint are set, it's weird? I think it's just saying that it needs to figure out the max connections either from the per instance/endpoint (which it calculates by multiplying by # instances/endpoints) or from the overall max. I'm in the middle of creating to verify, but I wanted to make sure what you meant.

Reply:
Ah yeah that makes sense. So what I was thinking was- the `max_connections` documentation says "For CONNECTION mode, either maxConnections or maxConnectionsPerInstance must be set", but that's not true- it could also be maxConnectionsPerEndpoint. So I think that for `max_connections` it should say all 3, and then for the instance/endpoint ones it should say something like "when the group is an instance group, MC or MCPI must be set"

Reply:
fair, most likely because the comment was probably written when perInstance was the only one - I'll edit it/file a bug against docs